dotnet-directory-sync-example/Controllers/HomeController.cs

```diff
@@ -121,6 +121,56 @@ public async Task<IActionResult> Group(string id, string directoryId)
             return View();
         }
 
+        [Route("users/{id?}")]
+        public async Task<IActionResult> Users(string id)
+        {
+            var serializeOptions = new JsonSerializerOptions { WriteIndented = true };
+
+            var directorySync = new DirectorySyncService();
+
+            var directory = await directorySync.GetDirectory(id);
+
+            // User logic
+            var userOptions = new ListUsersOptions
+            {
+                Directory = id,
+            };
+            var users = await directorySync.ListUsers(userOptions);
+
+            List<User> userList = new List<User>();
+            userList = users.Data;
+            ViewData["userList"] = userList;
+            ViewBag.Users = users;
+
+            return View();
+        }
+
+        [Route("groups/{id?}")]
```

Comment:
The groups aren't appearing here. It looks like there's an issue setting the directory ID as a query parameter based on what I can see in the URL. 

![CleanShot 2023-06-01 at 12 06 24](https://github.com/workos/dotnet-example-applications/assets/47123183/4046c859-1c5b-4740-b142-31280b316585)